workflow/scripts/retrieve_databundles.py

```diff
@@ -29,14 +30,18 @@ def download_repository(url, rootpath, repository):
     progress_retrieve(url, tarball_fn)
 
     logger.info(f"Extracting {repository} databundle.")
-    if (
-        repository == "EFS"
-    ):  # deflate64 compression not supported by zipFile, current subprocess command will only work on linux and mac
-        if platform.system() == "Windows":
-            cmd = ["tar", "-xf", tarball_fn, "-C", to_fn]
-        else:
-            cmd = ["unzip", tarball_fn, "-d", to_fn]
-        subprocess.run(cmd, check=True)
+    if repository == "EFS":
+        try:
+            # For Windows OS, use zipfile-deflate64
+            with zipfile_deflate64.ZipFile(tarball_fn, "r") as zip_ref:
+                zip_ref.extractall(to_fn)
+        except Exception:
+            # Fallback to system commands
+            if platform.system() == "Windows":
+                cmd = ["tar", "-xf", tarball_fn, "-C", to_fn]
```

Comment:
Is there any reason to keep this? Or can we just replace this with your code ... 

```python
with zipfile_deflate64.ZipFile(tarball_fn, "r") as zip_ref:
    zip_ref.extractall(to_fn)
```

... and just remove the `try` `except` block? That way Windows will always use `zipfile_deflate64`